sympy/series/tests/test_limits.py

```diff
@@ -698,6 +698,16 @@ def test_issue_14811():
     assert limit(((1 + ((S(2)/3)**(x + 1)))**(2**x))/(2**((S(4)/3)**(x - 1))), x, oo) == oo
 
 
+def test_issue_16222():
+    assert limit(exp(x), x, 10000000) == exp(10000000)
+    assert limit(exp(x), x, 100000000) == exp(100000000)
+    assert limit(exp(x), x, 1000000000) == exp(1000000000)
+
+
+def test_issue_16714():
+    assert limit(((x**(x + 1) + (x + 1)**x) / x**(x + 1))**x, x, oo) == exp(exp(1))
```

Comment:
This issue is already resolved and has a testcase. Please remove this testcase.

Reply:
I have made the changes. Actually this mentioned test was creating a conflict, I guess this was recently merged.